ui/src/subsonic/index.js

```diff
@@ -49,7 +49,9 @@ const getCoverArtUrl = (record, size) => {
     ...(record.updatedAt && { _: record.updatedAt }),
     ...(size && { size }),
   }
-  return baseUrl(url('getCoverArt', record.coverArtId || 'not_found', options))
+  if (record.coverArtId)
```

Comment:
Just a bit of nitpick: Can you put braces `{}` around the statements in the `if` and `else` blocks? Or rewrite it with a ternary operator:
```js
  return record.coverArtId
    ? baseUrl(url('getCoverArt', record.coverArtId, options))
    : baseUrl(url('getCoverArt', 'not_found', size && { size }))
```

I strive to avoid `if` blocks without braces throughout the code. This arguably makes code easier to maintain, reducing the possibility of wanting to add a new statement in the if block and forgetting to add the braces. This is similar to what Go forces you to do: https://golang.org/doc/effective_go#if

Reply:
No problem. Opt'd for braces in this case. The ternary seems a bit long and unweildy